public/js/action-list.js

```diff
@@ -106,6 +123,17 @@
                 url = $list.attr('data-icinga-multiselect-url') + '?(' + filters.toArray().join('|') + ')';
             }
 
+            if (! bottomStatusBar.find('.selection-count').length) {
+                bottomStatusBar.prepend('<div class="selection-count"></div>');
+            }
+
+            var label = $list.data('icinga-multiselect-count-label').replace('%d', $activeItems.length);
+            if (bottomStatusBar.find('.selection-count').find('.selected-items').length) {
```

Comment:
This would be a use-case for `find` now. If it would only be **one** `find` call, though. Please use `bottomStatusBar.find('.selection-count > .selected-items').length` instead, and cache it and re-use the result in the next line.